Plugin.cs

```diff
@@ -153,15 +148,71 @@ private void OnNetworkEvent(IntPtr dataPtr, ushort opCode, uint sourceActorId, u
             if (!this.configuration.alwaysOn && (!this.configuration.smart || !Retainer())) return;
             var listing = MarketBoardCurrentOfferings.Read(dataPtr);
             var i = 0;
-            if (this.configuration.hq && this.items.Single(j => j.RowId == listing.ItemListings[0].CatalogId).CanBeHq)
-            {
+
+            // Load addon ptr from memory
+            var toolTipPtr = pi.Framework.Gui.GetUiObjectByName("ItemDetail", 1) + 0x258;
+            var toolTipItemName = GetSeStringText(GetSeString(toolTipPtr));
+            bool isCurrentItemHQ = false;
+
+            // Checks for HQ symbol in item name
+            if (toolTipItemName.Substring(toolTipItemName.Length - 1) == "")
+                isCurrentItemHQ = true;
+            else
+                isCurrentItemHQ = false;
```

Comment:
Just make this `bool isCurrentItemHQ = toolTipItemName.Substring(toolTipItemName.Length - 1) == ""`

Reply:
Got it